packages/select/src/vaadin-select.js

```diff
@@ -344,6 +364,10 @@ class Select extends DelegateFocusMixin(FieldMixin(SlotMixin(ElementMixin(Themab
    * It is not guaranteed that the update happens immediately (synchronously) after it is requested.
    */
   requestContentUpdate() {
+    if (!this._overlayElement) {
+      return;
+    }
```

Comment:
This looks like an easy fix not directly related to this feature. Could you please submit a separate PR for it?

Reply:
Yeah, actually it was discovered by the unit tests I added for the `items` property.

Reply:
I'll extract it into a separate PR that would make sense to merge before this PR.

Reply:
https://github.com/vaadin/web-components/pull/2995

Reply:
Rebased this PR since the PR with the fix is merged.